src/asynk/async_queue.rs

```diff
@@ -268,18 +288,16 @@ where
 
     pub async fn update_task_state_query(
         transaction: &mut Transaction<'_>,
-        task: &Task,
+        task: Task,
```

Comment:
we don't have any tests for this query, let's add

Reply:
okay :D